jurity/recommenders/combined.py

```diff
@@ -106,8 +106,15 @@ def get_score(self, actual_results: pd.DataFrame, predicted_results: pd.DataFram
         """
         batch_vals = dict()
         acc_vals = dict()
+
+        for metric in self.metrics:
+            if ~isinstance(metric, _BaseRecommenders) and batch_accumulate:
```

Comment:
a quick one-liner explanation about why this is not allowed would be nice to have